parser.go

```diff
@@ -285,33 +299,60 @@ func (p *parser) readPythonMultilines(line string, bufferSize int) (string, erro
 	parserBufferPeekResult, _ := p.buf.Peek(bufferSize)
 	peekBuffer := bytes.NewBuffer(parserBufferPeekResult)
 
+	indentSize := 0
 	for {
 		peekData, peekErr := peekBuffer.ReadBytes('\n')
 		if peekErr != nil {
 			if peekErr == io.EOF {
+				p.debug("readPythonMultilines: failed to peek")
+				p.debug("readPythonMultilines: peekData is: '"+string(peekData)+"'")
+				p.debug("readPythonMultilines: value is: '"+line+"'")
 				return line, nil
 			}
+			p.debug("readPythonMultilines: failed to peek, returning error")
```

Comment:
Why this is necessary since `peekErr` is returned?

Reply:
Ping on this comment, possible missed.

Reply:
This is to provide an easy way to trace what exactly is failing and why. The debug() function should provide additional information beyond the thrown exception. When implementing debug logging, I usually have a debug log statement at every escape.

Reply:
OK, make sense, then I think probably could do this:
```suggestion
			p.debug("readPythonMultilines: failed to peek with error: %v", err)
```

Reply:
ok